bower.json

```diff
@@ -13,7 +13,7 @@
   },
   "devDependencies": {
     "requirejs": "~2.1.9",
-    "scribe": "guardian/scribe#master",
+    "scribe": "guardian/scribe#df035cf91b4033028f8562ebc0649d72a6abdbe8",
```

Comment:
The corresponding tag branch is built, which fails because noting requires the source version of Scribe.

The reason I'm doing this in the first place is because lodash is updated in Scribe, while Composer is a few Scribe versions behind and on lodash 2. I really just want to run the tests and see that they pass with the same version of Scribe that Composer is using.